build.gradle

```diff
@@ -655,10 +656,8 @@ if (project.hasProperty('mavenUser') && project.hasProperty('jcecertAlias')) {
             }
             println "Loaded staging id is " + stagingProperties['staging.id']
             stagingRepositoryId = stagingProperties['staging.id']
-        }
-        doFirst {
-            // TODO: Remove this once we want to automate releases
-            ant.fail("Release from gradle is currently disabled")
```

Comment:
Don't forget that this is also commented out in the buildspec too.

https://github.com/corretto/amazon-corretto-crypto-provider/blob/21172da62e81e97e56fa15691ae8dbe59b7d450c/tests/ci/codebuild/release/release.yml#L36-L39

Reply:
The release pipeline is implemented in CDK and maintained in an internal repo. The pipeline does not use the build specs defined here. In the CDK code, the line that invokes Gradle with `releaseSonatypeStagingRepository` is uncommented.

Reply:
That was my guess. We should probably remove the old specs then. (That's a better way to handle it anyway.)